saleor/graphql/middleware.py

```diff
@@ -27,3 +30,27 @@ def middleware(request):
         return get_response(request)
 
     return middleware
+
+
+def get_service_account(auth_token):
```

Comment:
I think it would be useful to hint `Optional[ServiceAccount]` so we directly know it returns `None` if it's not found 👍